platform_umbrella/apps/verify/test/forgejo_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Verify.ForegejoTest do
+  use Verify.TestCase, async: false, batteries: ~w(forgejo)a
+
+  verify "forgejo is running", %{session: session} do
+    session
+    |> assert_pod_running("pg-forgejo-1")
+    |> assert_pod_running("forgejo-0")
```

Comment:
Is this a good time to add an `assert_can_go_to_web_UI_linked_here` tool?

Reply:
I was just debating that. I'll go ahead and add it for the next one and back fill these last few that could use it.